libs/design-system/logos/apps/src/lib/app-logos.component.scss

```diff
@@ -0,0 +1,116 @@
+:host {
+  display: flex;
+  height: 2.375rem;
+
+  &[variant='fixed'] {
+    height: 3rem;
+
+    .hra-logo {
+      margin-right: 0.25rem;
+      width: unset;
+
+      .hra-logo-text {
+        display: none;
+      }
+    }
+
+    .app-logo {
+      img {
+        margin-right: 0.75rem;
+      }
+
+      .app-label {
+        font-size: 1.25rem;
```

Comment:
Use the appropriate `--sys-*` css variable for font and letter-spacing. Check the tooltip-card component for example usage